src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -108,15 +146,19 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
       }
       onHide();
     },
-    [onHide, eventSpaceId, eventSpace, event]
+    [
+      eventSpaceId,
+      selectedSpace.id,
+      eventSpace?.worldId,
+      worldId,
+      onHide,
+      event?.id,
+    ]
   );
 
-  const showDeleteButton = event?.id;
-  const handleDelete = () => {
-    onHide();
-    setEditedEvent && setEditedEvent(event);
-    setShowDeleteEventModal();
-  };
+  if (isSpacesLoading) {
+    return <div>loading</div>;
```

Comment:
Can some of the `<LoadingPage />` or `<Loading />` components be used here? If they're attendee side only, we may need to ping Design for update on this